content/en/blog/_posts/2022-08-01-csi-migration-status.md

```diff
@@ -0,0 +1,110 @@
+When a Kubernetes cluster administrator updates a cluster to enable CSI migration, existing workloads that utilize PVCs which are backed by in-tree storage plugins will continue to function as they always have.
+However, behind the scenes, Kubernetes hands control of all storage management operations (previously targeting in-tree drivers) to CSI drivers.
+
+For example, suppose you are a `kubernetes.io/gce-pd` user, after CSI migration, you can still use `kubernetes.io/gce-pd` to provision new volumes, mount existing GCE-PD volumes or delete existing volumes. All existing API/Interface will still function correctly. However, the underlying function calls are all going through the [GCE PD CSI driver](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) instead of the in-tree Kubernetes function.
+
+This enables a smooth transition for end users. Additionally as storage plugin developers, we can reduce the burden of maintaining the in-tree storage plugins and eventually remove them from the core Kubernetes binary.
+
+## What is the timeline / status? {#timeline-and-status}
+
+The current and targeted releases for each individual driver is shown in the table below:
+
+| Driver           | Alpha         | Beta (in-tree deprecated) | Beta (on-by-default) | GA            | Target "in-tree plugin" removal |
+| ---------------- | ------------- | ------------------------- | -------------------- | ------------- | ------------------------------- |
+| AWS EBS          | 1.14          | 1.17                      | 1.23                 | 1.25          | 1.27 (Target)                   |
+| GCE PD           | 1.14          | 1.17                      | 1.23                 | 1.25          | 1.27 (Target)                   |
+| OpenStack Cinder | 1.14          | 1.18                      | 1.21                 | 1.24          | 1.26 (Target)                   |
+| Azure Disk       | 1.15          | 1.19                      | 1.23                 | 1.25          | 1.27 (Target)                   |
+| Azure File       | 1.15          | 1.21                      | 1.24                 | 1.26 (Target) | 1.27 (Target)                   |
+| vSphere          | 1.18          | 1.19                      | 1.25                 | 1.26 (Target) | 1.28 (Target)                   |
+| Ceph RBD         | 1.23          | 1.26 (Target)             | 1.27 (Target)        | 1.28 (Target) | 1.30 (Target)                   |
+| Portworx         | 1.23          | 1.25                      | 1.26 (Target)        | 1.27 (Target) | 1.29 (Target)                   |
+| CephFS           | 1.26 (Target) |                           |                      |               |                                 |
+
+The following storage drivers will not have CSI migration support.
+The `ScaleIO`, `Flocker`, `Quobyte` and `StorageOS` drivers were removed; the others are deprecated and will be removed from core Kubernetes in the coming releases.
```

Comment:
If writing these in backticks, please use the volume driver name that you'd use in a manifest (eg `quobyte`).

Reply:
Done